### ISSUE TITLE
Unable to create Shop for ConcretePowder:11

### ISSUE THREAD

**Issue by smmmadden** [KaiKikuchi/QuickShop#138]
I was able to add shop signs for all chests but this one.  It wasn't a max limit reached as I tried this several times and only get a "Cancelled Shop Purchase" with no explanation what's wrong.  All the other signs were created with Crouch and left click with the item, just this one fails for some reason.  Spigot 1.12/BungeeCord Server 1.12 and QuickShop 0.9.34.
![image](https://user-images.githubusercontent.com/10309346/27505762-f3c3acd0-5875-11e7-94af-019fc6b54fa0.png)
Thanks, 
Steve

**Comment by KaiKikuchi** [comment]
It's not because it's ConcretePowder:11, the plugin definitely doesn't behave differently if that's the item.

From what I see, the shop is being created but the sign doesn't appear. Just put an empty sign manually, then click it.

This is usually caused because the block where the sign would be placed into is occupied by something that is not a very empty block. Although you do not see any block there, the server says that block is not empty and won't place a sign for you. You can place an empty sign manually, then click it. The plugin will automatically detect the sign and write the necessary stuff. Also notice that if in front of the shop chest there is another shop chest, then it may not work.

**Comment by smmmadden** [comment]
You are right, it did think something was blocking it.  There was a torch initially and I removed/broke it and then tried to create the shop which "Cancelled Shop Purchase" shows up.  If it thought there was a block preventing it from being placed, is there a way to show "Cancelled, block occupies space" or something more helpful to the player? 
So I logged back in, and placed a chest in the spot occupied by "nothing" and it placed fine.
![image](https://user-images.githubusercontent.com/10309346/27508719-5a321234-58b9-11e7-8860-e90c0ee8e2de.png)
I broke the chest and now I was able to add the shop.  Weird that it thought something was there, but I could put something else in its place. 
I also see that somehow today, the name of the powder is shorter for a few of the same powder blocks but I haven't changed anything.
![image](https://user-images.githubusercontent.com/10309346/27508733-af4e98c8-58b9-11e7-8ffe-95431e396f74.png)

**Comment by KaiKikuchi** [comment]
The shop is being create anyway even if the sign doesn't appear. You can notice that because if you click the shop, it'll show the menu like any other shop. In that case, you really just have to remove anything on the block where the sign needs to be, and eventually add an empty sign manually. The message "shop *purchase* cancelled" is shown because there's already a shop there.

About the ConcretePowder issue, just click on all the signs or open/close the shop chests, they signs will update with the new item name. This probably happened because you set the new name on the config file.

**Comment by smmmadden** [comment]
Thanks Kai, makes sense but there wasn't a shop created as the putting the price ends up with the shop being cancelled.  I did try multiple times, but it was only when I placed a solid block in front of the chest and then remove it, that I was able to complete the creation of the shop.  I at least now know what can be done to get around it, it was just annoying b/c the "Cancelled Shop Purchase" wasn't telling me why it was cancelled.

To the point of the Powder blocks, it appears to only be abbreviated for ConcretePowder:10 through ConcretePowder:15 blocks.  The others are not abbreviated even trying to update the sign as suggested.  So it appears like the double digit blocks for this new item is slightly different than the single digit ones.
